### DISCUSSION TITLE
List of binaries

### DISCUSSION THREAD

**Discussion by eddumelendez** [microsoft/openjdk#36] [Q&A] [answered]
Currently, I am able to see the latest versions of microsoft open jdk [here](https://www.microsoft.com/openjdk) but in this case what if I am looking for previous versions, is there any site I can look for this? I would like to submit the previous jdk 16.0.0 for aarch64 to sdkman in order to list all the versions available.

**Comment by karianna** [comment, marked as answer]
Hi @eddumelendez - We intend to publish a single archive (API) of our binaries in the not too distant future. However, the 16.0.0 (16+36) builds were Early Access (preview) and so we have not yet decided if we will make those available via that future archive/API.

For folks who need access to those historical binaries today, you can get them from https://github.com/microsoft/openjdk-aarch64/releases. 
 Please note that location should not be considered a permanent URL.
